experimental/PropertyDDS/packages/property-binder/package.json

```diff
@@ -42,8 +56,8 @@
 		"test": "npm run test:jest",
 		"pretest:coverage": "rimraf --glob coverage/",
 		"test:coverage": "c8 npm test",
-		"test:jest": "jest",
-		"tsc": "tsc"
+		"test:jest": "jest --passWithNoTests",
```

Comment:
added --passWithNoTests

Reply:
There were no tests because the tsconfig.json settings were changed and removed "allowJs". Restored and now there are tests again. :)